pkg/bloombuild/planner/strategies/splitkeyspace/strategy.go

```diff
@@ -0,0 +1,245 @@
+package splitkeyspace
```

Comment:
nit: Is is really necessary to make a separate package for each individual strategy?